src/platform/site-wide/user-nav/helpers.js

```diff
@@ -0,0 +1,33 @@
+import { fromPairs } from 'lodash';
+
+import { hasSession, hasSessionSSO } from 'platform/user/profile/utilities';
+import { ssoKeepAliveSession, getForceAuth } from 'platform/utilities/sso';
+import { autoLogin, autoLogout } from 'platform/user/authentication/utilities';
+
+function parseqs(value) {
+  /*
+   * naive query string parsing function, takes a query string and returns
+   * an object mapping the keys to values.
+   */
+  const data = value.startsWith('?') ? value.substring(1) : value;
+  const entries = data ? data.split('&').map(q => q.split('=')) : [];
+  return fromPairs(entries);
+}
```

Comment:
We have a module `url-search-params` that should make this easier. I think it's actually a backfill for `window.URLSearchParams`.